san/get.py

```diff
@@ -16,4 +30,14 @@ def get(dataset, **kwargs):
     gql_query = "{\n" + gql_query + "\n}"
     res = execute_gql(gql_query)
 
+    if query in QUERY_PATH_MAP:
+        res = shorten_path(res, QUERY_PATH_MAP[query])
+        if query == 'eth_top_transactions':
```

Comment:
This is a special case and is awkward to have it in the generic `get` function. Currently I think we do not have a better place to place it be we need to implement it.

Reply:
Possible solution is to be able to attach transform functions to the queries that will run before the result is returned.   Having by default the idempotent function (`id(x) = x`) and only changing where it's needed is a more extendable way.

Reply:
That being said, if we have such transform functions that work in a generic way they can be executed here. Pseudo code:
```python
if query in QUERY_PATH_MAP:
   res = shorten_path(res, QUERY_PATH_MAP[query])
   transform_fn = get_transform_fn(query)
   apply transform_fn to res
```